docs/source/community/documentation/DocumentationStyle.md

```diff
@@ -0,0 +1,129 @@
+
+## What is documentable?
+Evennia's core documentation is split into the following key areas:
+- Core/Evennia API
+- Tutorials and Examples
+- Community
+- Related Topics
+- Full API Reference
+
+### Core/Evennia API
+This section of the documentation is a game-agnostic tutorial for using specific structures available in the Evennia flat API.
+
+The following types of information would be relevant here:
+- Documentation of the features available in the Typeclass system
+- Documentation about the Message Path and OOB communications
+- Glossaries of Evennia terms, like what a "Puppet" is
+
+The following types of information would be out of place:
+- A tutorial on building a 'clothing' system. This is not game-agnostic; it would be more appropriate in the Tutorials section
+- Documentation related to Apache/Nginx configuration. This is not an Evennia concept, this would belong in the Server section
+
+### Tutorials and Examples
+This section deals with specific systems that users can implement in their game. 
+
+The following would be appropriate here:
+
+- Documentation for a contrib
+- A tutorial on how to build a 'clothing' system
+- A tutorial on adding a Wikipedia page to your website
+
+### Community
+This section of the documentation surrounds the Evennia community. It includes subsections related to
+the future of the Evennia project. Opinions also live here.
+
+The following would be appropriate here:
+
+- How Evennia's core repository uses Continual Integration
+- How to get and give help in Evennia; official channels
+- What we think Evennia is, and how we think it should be used
+- A discussion of the potential for refactoring Evennia submodules
+
+### Related Topics
+This section of the documentation deals with the pieces of Evennia that are technically 'outside' of Evennia.
+
+The following topics would be appropriate here:
+
+- How to use Evennia to learn Python
+- How to administer a production server running Evennia
+
+### Full API Reference
+This section is the technical documentation for Evennia. It is generated automatically
+from the source of Evennia. It provides a detailed reference for specific modules.
+
+## Formatting Documentation
+The Evennia documentation uses Markdown formatting and the conventions defined in the Wikipedia
+Manual of Style.
+
+### Table of Contents
+A table of contents should be present on each page of the documentation. This is generated
+automatically when the documentation is built.
+
+### Disambiguation
+Disambiguation documents should be created to help redirect people who might search
+the documentation. These pages are flagged with the following source code:
+
+```python
+TYPE = "Disambiguation"
+```
+
+Disambiguation pages should also be created when links are broken and pages renamed or move. 
+
+### Rating
+
+All pages receive a **rating** in the documentation. 
+
+```python
+RATING = "Exceptional"
+```
+
+The following ratings are available.
```

Comment:
Ah, you're missing the key communication (which really hasn't been communicated well yet).

Exceptional-quality documentation is exceptional because the community _makes_ it exceptional.

It's designed to be the must-reads, that makes it the most-read. That makes it the most edited. That means if you see something wrong with it, definitely fix it. But maybe don't add a long paragraph about some random use case that you found; if you do, it may not get merged.

On the other hand, Needs Improvement-quality documentation is literally marked as such because we can expect to accept pretty much any changes to the documentation that are reasonable. If you think you want to put a completely different spin on it, and it's Needs Improvement, that WOULD probably get merged.

It's really a classification of how much scrutiny your changes will get, rather than a classification of whether or not you should try.

It also means that you can keep a list of a half-dozen must-read "exceptional" level documents that you regularly check. You know, like how your "installing evennia" still used python2 virtualenv, even though we've been off python2 for a long time? Feels like if that was one of the pieces of documentation that was regularly reviewed, that would have been caught right away.

Reply:
I understand how you _intend_ the rating to be perceived, it's not how it will _be_ perceived. Most people will not be reading the instructions for what these ratings mean. And even so, it still comes across as very self-grandizing and arrogant to refer to itself as "Rating=Excellent" with no further qualifier. It just gives the wrong connotations. And yes, people will get discouraged from contributing by it. 

That said, it's not a bad idea to have something to track the important parts of the documentation. So maybe a rating `IMPORTANCE` or maybe `MAINTENANCE-IMPORTANCE` or something like that, with a rating meant for documentation-maintainers to be able to keep track of which documents really needs attention more regularly than others? 

### MAINTENANCE-IMPORTANCE

- Low
- Medium
- High
- Critical

### COMPLETENESS 

- Unknown
- Needs Improvements
- Complete

### COMPLEXITY 

- Beginner
- Intermediary
- Advanced 

> Btw: I still daily use virtualenv the same way as described in the installing evennia page; it works fine for python3, so changing that or not had nothing to do with with not giving the page attention ... ahem.